src/pages/trade-types/multiplier/_synthetic-indices.js

```diff
@@ -166,6 +166,13 @@ const SyntheticIndices = () => {
                     />
                 </MarketsWrapper>
                 <AvailablePlatforms dtrader />
+                <Show.Eu>
+                    {synthetic_multiplier.eu_content?.map((text, index) => (
```

Comment:
Everything looks good~ Only a question, will there be any instance that eu_content might return null or undefined?

Reply:
It won't be null or undefined unless someone renames the key of remove it. It's the same content used in two different pages.

Reply:
If that's the case, i think it's better if we remove the optional chaining here since the value is static, what do you think? 😄

Reply:
Sounds good to me. Resolved.